scheduler/local_scd_server.py

```diff
@@ -243,15 +244,23 @@ def parse_swg_to_scd(self, modes, radar, first_run):
 
 def main():
     """ """
-    parser = argparse.ArgumentParser(description="Automatically schedules new events from the SWG")
-    parser.add_argument('--scd-dir', required=True, help='The scd working directory')
-    parser.add_argument('--force', action="store_true", help='Force an update to the schedules '
-                                                             'for the next month')
-    parser.add_argument('--first-run', action="store_true", 
-                        help='This will generate the first set of schedule files if running on'
-                             ' a fresh directory. If the next month schedule is available,'
-                             ' you will need to roll back the SWG schedule folder back to the'
-                             ' last commit before running in continuous operation.')
+    parser = argparse.ArgumentParser(
+        description="Automatically schedules new events from the SWG"
+    )
+    parser.add_argument("--scd-dir", required=True, help="The scd working directory")
+    parser.add_argument(
+        "--force",
+        action="store_true",
+        help="Force an update to the schedules " "for the next month",
+    )
```

Comment:
Why are some changed to multi-line, but not all?

Reply:
The line must've been too long so it got wrapped